src/Microsoft.Graph.Core/Properties/AssemblyInfo.cs

```diff
@@ -30,7 +30,11 @@
 #if DEBUG
 [assembly: InternalsVisibleTo("Microsoft.Graph.Core.Test")]
 [assembly: InternalsVisibleTo("Microsoft.Graph.Test")]
+[assembly: InternalsVisibleTo("Microsoft.Graph.DotnetCore.Core.Test")]
+[assembly: InternalsVisibleTo("Microsoft.Graph.DotnetCore.Test")]
 #else
 [assembly: InternalsVisibleTo("Microsoft.Graph.Core.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
 [assembly: InternalsVisibleTo("Microsoft.Graph.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
+[assembly: InternalsVisibleTo("Microsoft.Graph.DotnetCore.Core.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
The fully-qualified assembly names should be the same regardless of what target you are compiling against. This allows for things like bait-and-switch style reference, where you reference .NET Standard at compile time, but end up with a special .NET 4.5 version at runtime.